rotkehlchen/balances/manual.py

```diff
@@ -47,7 +47,7 @@ def get_manually_tracked_balances(db: 'DBHandler') -> List[ManuallyTrackedBalanc
 
         balances_with_value.append(ManuallyTrackedBalanceWithValue(
             **entry._asdict(),
-            usd_value=price * entry.amount,
```

Comment:
Why are you introducing this change here? Price should **always** be an FVal here.

Reply:
I made this test fail https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/api/test_balances.py#L140 in multiple occasions in the develop branch with an error related to FVal operation

Reply:
This is not the right place to fix as discussed in discord. Please revert this. It should be obvious to you from the types. The price should always already be an `FVal` here. So the problem is not here, but in whatever gave you the price. I am fixing this in develop.